x-pack/plugins/security_solution/public/cases/pages/translations.ts

```diff
@@ -10,3 +10,17 @@ import { i18n } from '@kbn/i18n';
 export const PAGE_TITLE = i18n.translate('xpack.securitySolution.cases.pageTitle', {
   defaultMessage: 'Cases',
 });
+
+export const ISOLATED_HOST = i18n.translate('xpack.securitySolution.caseView.isolatedHost', {
+  defaultMessage: 'submitted isolate request on host',
```

Comment:
Thanks I have already tried that :) Unfortunately we cannot import from another plugin (security_solution vs cases).

Reply:
Actually we have been able to import from other plugins since a while now using `@kbn/<pluginname>-plugin/<exportpath>`. Try this:

```json5
import {
  ISOLATED_HOST,
  RELEASED_HOST,
} from '@kbn/cases-plugin/public/components/user_actions/translations';
```

Reply:
Not really, I think we were able to import types, but not actual code: 
``` ERROR in ./public/cases/attachments/external_reference_event.tsx
       │          Module not found: Error: import [@kbn/cases-plugin/public/components/user_actions/translations] references a non-public export of the [cases] bundle and must point to one of the public directories: [public,common]```

Reply:
I think this is why @kbn/[package] is used for

Reply:
Thanks for trying it out. The error message tells you that you need to export those variables from a public/common directory, not that you can't import from cases plugin. You also need to export these variables from 
https://github.com/elastic/kibana/blob/3ef1e76667a9473c1bf31b2cc7176c4f8311ea54/x-pack/plugins/cases/public/index.tsx#L25

You'll have to add 
```json5
export { ISOLATED_HOST, RELEASED_HOST } from './components/user_actions/translations';
```
to the above file and then you should be able to import like so:

```json5
import { ISOLATED_HOST, RELEASED_HOST } from '@kbn/cases-plugin/public';
```

See how `getCasesDeepLinks` is imported from cases in https://github.com/elastic/kibana/blob/3ef1e76667a9473c1bf31b2cc7176c4f8311ea54/x-pack/plugins/security_solution/public/cases/links.ts#L13